docker-run.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-INDEX_FILE=/usr/share/nginx/html/index.html
+INDEX_FILE=/usr/share/nginx/html/${BASE_PATH}index.html
```

Comment:
What do you mean by internal purposes ?

I would like to deploy a swagger service behind a reverse proxy.  So if the reverse proxy URI to access swagger is http://reverse-proxy-host/swagger, I need to have the same URI in docker nginx, otherwise the ProxyPass directives will not work.

ProxyPass /swagger http://docker-host:docker-port won't work
ProxyPass /swagger http://docker-host:docker-port/swagger will work

This is why I added BASE_PATH